Вынести весь функционал связанный с tasks в отдельную библиотеку

sinelamin opened the pull request.

Commit:
feat(tasks): вынести весь функционал связанный с tasks в отдльную бибилиотеку

В библиотеку включены:
Модуль tasks и компоненты tasks-list, task-item
Сервис и интерфейс
Модуль реэкспорта с импортам из Ангуляр Материал

tamazyanarsen commented on src/app/feature/todo-input/todo-input.module.ts:
```diff
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { ReactiveFormsModule } from '@angular/forms';
 
-import { AngularMaterialModule } from 'src/app/shared/angular-material.module';
+import { AngularMaterialModule } from 'projects/task/src/public-api';
```
импорт должен быть из названия библиотеки, которую ты зарегистрировал в angular.json и tsconfig

`task`

sinelamin committed:
rafactor(task): заменить относительные пути импортов к библиотеки task

Заменил в импортах относительные пути к библиотеке на пути к собранному билду task
